demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/05/Version20240513094920.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Exception;
+use Doctrine\DBAL\Platforms\MySQLPlatform;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+class Version20240513094920 extends AbstractMigration
```

Comment:
Error on execution:
`An exception occurred while executing a query: SQLSTATE[HY000]: General error: 1005 Can't create table `blp_release`.`access_control_permissions` (errno: 150 "Foreign key constraint is incorrectly formed")`
Not sure at all if this may be my bad, need

Reply:
Updated in new version, also migration updated demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/05/Version20240521084035.php